webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const formatSourceLink = (hostAttrs, { currentValue }) => {
+  const value = `${currentValue.element}: ${currentValue.elementName}`;
+  let { element } = currentValue;
+  const { meta } = currentValue;
+
+  const knownMatchResources = ['os', 'hostgroup', 'domain'];
```

Comment:
Why only these three and not other resources?

Reply:
These are the values offered in 'Prioritize attribute order' when editing. There are too many special cases that cannot be reliably handled in a generic way

1. Since matchers can be set up on virtually anything, there is no way to tell if we match on host attribute (such as comment) or on associated resource.
2. Some resources may may have naming mismatches similar to operating system
3. Katello has a very different page paths than Foreman and there is no way to tell where the resource comes from
4. Some resources extend host via facets

Reply:
> These are the values offered in 'Prioritize attribute order' when editing.

If I'm not mistaken, the default order is defined by a setting, and every role can have it's own override order. 
I agree matchers could be set on anything and it's hard to guess the correct path - perhaps as a first step we could drop the attempt to link to the resource and only display the type and name? Then we also don't need to check the `canEdit` permission